inference-engine/ie_bridges/python/sample/speech_sample/speech_sample.py

```diff
@@ -137,15 +137,21 @@ def main():
         else:
             utterances = read_utterance_file(args.input.split(',')[0])
             key = sorted(utterances)[0]
-            scale_factor = get_scale_factor(utterances[key])
-            log.info(f'Using scale factor of {scale_factor:.7f} calculated from first utterance.')
-
-            plugin_config['GNA_SCALE_FACTOR'] = str(scale_factor)
+            if args.scale_factor:
+                log.info('Using user defined scale factor of %s', str(args.scale_factor))
+                plugin_config['GNA_SCALE_FACTOR'] = str(args.scale_factor)
+            else:
+                scale_factor = get_scale_factor(utterances[key])
+                log.info(f'Using scale factor of {scale_factor:.7f} calculated from first utterance.')
+                plugin_config['GNA_SCALE_FACTOR'] = str(scale_factor)
 
         if args.export_embedded_gna_model:
             plugin_config['GNA_FIRMWARE_MODEL_IMAGE'] = args.export_embedded_gna_model
             plugin_config['GNA_FIRMWARE_MODEL_IMAGE_GENERATION'] = args.embedded_gna_configuration
 
+        if args.performance_counter:
+            plugin_config['PERF_COUNT'] = 'YES' 
+
```

Comment:
Why is the option enabled only for GNA device?

Reply:
I believe this aligns with the current C++ implementation.